src/main/java/org/broadinstitute/hellbender/utils/clipping/ClippingOp.java

```diff
@@ -50,34 +50,34 @@ GATKRead apply(final ClippingRepresentation algorithm, final GATKRead originalRe
             //   because you're not guaranteed to get a pointer to the actual array of bytes in the GATKRead
             case WRITE_NS: {
                 final GATKRead readCopied = originalRead.copy();
-                applyWRITE_NS(readCopied);
+                applyWriteNs(readCopied);
                 return readCopied;
             }
 
             case WRITE_Q0S: {
                 final GATKRead readCopied = originalRead.copy();
-                applyWRITE_Q0S(readCopied);
+                applyWriteQ0s(readCopied);
                 return readCopied;
             }
 
             case WRITE_NS_Q0S: {
                 final GATKRead readCopied = originalRead.copy();
-                applyWRITE_NS(readCopied);
-                applyWRITE_Q0S(readCopied);
+                applyWriteNs(readCopied);
+                applyWriteQ0s(readCopied);
                 return readCopied;
             }
 
             case HARDCLIP_BASES: {
                 //Note: passing the original read here because the read is copied right away inside the method
-                return applyHARDCLIP_BASES(originalRead, start, stop);
+                return applyHardclipBases(originalRead, start, stop);
             }
 
             case SOFTCLIP_BASES: {
-                return applySOFTCLIP_BASES(originalRead.copy(), runAsserts);
+                return applySoftClipBases(originalRead.copy(), runAsserts);
```

Comment:
It seems like this should be applySoftclipBases (lower-c) based on the location of underscores and the spelling of applyHardclipBases. In either case, the "c" should be the same case in those two functions.